katello/katello-change-hostname

```diff
@@ -237,6 +238,12 @@ def hammer_cmd(cmd, exit_codes=[0], message=nil)
   run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
 end
 
+def get_default_proxy_id
+  output = hammer_cmd("--output json capsule list --name #{@old_hostname}")
```

Comment:
Here `list` should be `info`,  with capsule list hammer says `Error: Unrecognised option '--name'`

Reply:
nevermind :)